package.json

```diff
@@ -106,21 +107,21 @@
     "lint-build": "node ./scripts/rollup/validate/index.js",
     "extract-errors": "yarn build --type=dev --extract-errors",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json && node ./scripts/flow/createFlowConfigs.js && node ./scripts/yarn/downloadReactIsForPrettyFormat.js",
-    "debug-test": "cross-env NODE_ENV=development node --inspect-brk node_modules/jest/bin/jest.js --config ./scripts/jest/config.source.js --runInBand",
-    "test": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.source.js",
-    "test-www": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.source-www.js",
-    "test-www-variant": "cross-env NODE_ENV=development VARIANT=true jest --config ./scripts/jest/config.source-www.js",
-    "test-prod-www": "cross-env NODE_ENV=production jest --config ./scripts/jest/config.source-www.js",
-    "test-prod-www-variant": "cross-env NODE_ENV=production VARIANT=true jest --config ./scripts/jest/config.source-www.js",
-    "test-persistent": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.source-persistent.js",
-    "debug-test-persistent": "cross-env NODE_ENV=development node --inspect-brk node_modules/jest/bin/jest.js --config ./scripts/jest/config.source-persistent.js --runInBand",
-    "test-prod": "cross-env NODE_ENV=production jest --config ./scripts/jest/config.source.js",
-    "debug-test-prod": "cross-env NODE_ENV=production node --inspect-brk node_modules/jest/bin/jest.js --config ./scripts/jest/config.source.js --runInBand",
-    "test-prod-build": "yarn test-build-prod",
-    "test-build": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.build.js",
-    "test-build-prod": "cross-env NODE_ENV=production jest --config ./scripts/jest/config.build.js",
-    "test-build-devtools": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.build-devtools.js",
-    "debug-test-build-devtools": "cross-env NODE_ENV=development node --inspect-brk node_modules/jest/bin/jest.js --config ./scripts/jest/config.build-devtools.js",
+    "debug-test": "yarn test --debug",
+    "test": "node ./scripts/jest/jest-cli.js",
+    "test-www": "yarn test --project www",
+    "test-www-variant": "yarn test --project www --variant",
+    "test-prod-www": "yarn test --prod --project www ",
+    "test-prod-www-variant": "yarn test --prod --project www --variant",
```

Comment:
Super trivial nit that I expect you to ignore *but*

Especially when we're mixing flags and enums, I think explicit "=" for enums make the command easier to visually parse.

Reply:
Thanks for the feedback, I went back and forth on this a few times, will update!